examples/azure-rules.yaml

```diff
@@ -656,4 +656,51 @@ groups:
           Please, check the naming convention in case the resource configuration is done correctly.
            
           More information at https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/sap-hana-high-availability#create-sap-hana-cluster-resources
+        scored: true
+  - id: 1.6
+    description: "Check OS and package versions are supported and up-to-date"
+    checks:
+      - id: 1.6.1
+        description: "Check os release"
+        audit: 'cat /etc/os-release'
+        tests:
+          bin_op: or
+          test_items:
+            - flag: 'PRETTY_NAME="SUSE Linux Enterprise Server for SAP Applications 12 SP4"'
+            - flag: 'PRETTY_NAME="SUSE Linux Enterprise Server for SAP Applications 15"'
+            - flag: 'PRETTY_NAME="SUSE Linux Enterprise Server for SAP Applications 15 SP1"'
+            - flag: 'PRETTY_NAME="SUSE Linux Enterprise Server for SAP Applications 15 SP2"'
+            - flag: 'PRETTY_NAME="SUSE Linux Enterprise Server for SAP Applications 15 SP3"'
+        remediation: |
+          Install a proper OS release
+        scored: true
+      - id: 1.6.2
+        description: "Check pacemaker version"
+        audit: 'rpm -qv pacemaker | sed -e "s/pacemaker-\(.*\)+.*/\1/"'
+        tests:
+          bin_op: or
+          test_items:
+            - flag: '2.0.3'
+        remediation: |
+          Update the pacemaker
+        scored: true
+      - id: 1.6.3
+        description: "Check crosync version"
+        audit: 'rpm -qv corosync'
+        tests:
+          bin_op: or
+          test_items:
+            - flag: 'corosync-2.4.5-5.8.x86_64'
```

Comment:
is there a "not in" combination? that would be more stable for the future, by simply listing what are *outdated* versions that we do not want to see. otherwise the next mainteannce update will break this check..

Reply:
I have added excluding rules for the pacemaker and sbd, but they exclude now valid releases, pacemaker-2.0.3+20200511.2b248d828 and sbd-1.4.0+20190326.c38c5e6.